admin/packaging.py

```diff
@@ -754,6 +764,14 @@ class PACKAGE_NODE(PACKAGE):
     )
 
 
+class PACKAGE_DOCKER_PLUGIN(PACKAGE):
+    DESCRIPTION = ValueConstant(
+        'Volume plugin for Docker\n'
+        + fill('This meta-package contains links to the Flocker Docker plugin',
+               79)
```

Comment:
Updated, and the other packages to be more consistent with our other marketing descriptions.